mingw-w64-curl/PKGBUILD

```diff
@@ -9,7 +9,7 @@ pkgdesc="Command line tool and library for transferring data with URLs. (mingw-w
 arch=('any')
 url="https://curl.haxx.se"
 license=("MIT")
-makedepends=("${MINGW_PACKAGE_PREFIX}-gcc" "${MINGW_PACKAGE_PREFIX}-pkg-config")
+makedepends=("${MINGW_PACKAGE_PREFIX}-cc" "${MINGW_PACKAGE_PREFIX}-pkg-config")
 depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
```

Comment:
Do we have a `gcc-libs` package for `CLANGARM64`?

Reply:
Yes, it's provided by [`mingw-w64-clang-aarch64-libc++`](https://packages.msys2.org/package/mingw-w64-clang-aarch64-libc++). Probably some compatibility solution by the MSYS2 team. [It's used in the MSYS2's curl package as well](https://github.com/msys2/MINGW-packages/blob/167f4294e8440b9513136022c1396f74768b9858/mingw-w64-curl/PKGBUILD#L26).